db/data/20190418114011_initialise_state_for_users.rb

```diff
@@ -0,0 +1,15 @@
+class InitialiseStateForUsers < ActiveRecord::Migration[5.2]
+  def up
+    User.find_each do |user|
+      user.save!
+    end
+  end
+
+  def down
+    User.find_each do |user|
+      next unless user.new?
+
+      user.update(aasm_state: '')
```

Comment:
Not sure this is right. There will always be an initial state set in the model i.e. `state :new, initial: true`

Reply:
If we do that then it's not a full reverse-migration; the initial database state is empty string. 🤔